content/en/profiler/enabling/ddprof.md

```diff
@@ -178,30 +178,8 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/tmp/ddprof/lib
 
 ## Configuration
 
```

Comment:
This table is being removed as it is redundant compared to the link with the versioned documentation we are providing.
The documentation is always up to date in `ddprof` github repository.